internal/graph/lookupsubjects.go

```diff
@@ -111,6 +111,9 @@ func (cl *ConcurrentLookupSubjects) lookupDirectSubjects(
 		if it.Err() != nil {
 			return it.Err()
 		}
+		if tpl.Caveat != nil {
```

Comment:
choke point to prevent evaluation of the graph if relationship is caveated. This is a temporal measure until caveat support is introduced in `Lookup*` methods